frontend/src/components/Collapsible.js

```diff
@@ -0,0 +1,36 @@
+import * as React from "react";
+import arrowImage from "../images/chevron-right.png";
+import { useState } from "react";
+import arrowImage2 from "../images/chevron-down.png";
+
+import {
+    arrow,
+    subHead,
+    box,
+} from "../css/belief.module.css"
+
+
+export const Collapsible = (props) => {
```

Comment:
Also (this is moot if we don't end up needing collapsibles), you can give more descriptive names for property members directly in the function signature like so:

```
export const Collapsible = ({sectionText, sectionTitle}) => {
   return (
   ...
        <div className={box}>
            <div className={subHead}>
                <h3>{sectionText}</h3>
            </div>
        </div>
    );
}
```

In general we should avoid names like prop1 and prop2